src/prefect/tasks/google/storage.py

```diff
@@ -256,3 +256,113 @@ def run(
         gcs_blob = self._get_blob(bucket, blob, encryption_key_secret)
         gcs_blob.upload_from_string(data)
         return gcs_blob.name
+
+
+class GCSCopy(Task):
+    """
+    Task template for copying data from one Google Cloud Storage blob to another, without
+    downloading it locally.
+
+    Args:
+        - source_bucket (str, optional): default source bucket name; can be overwritten at
+            runtime. Required prior to running the task.
+        - source_blob (str, optional): default source blob name; can be
+            overwritten at runtime.  Required prior to running the task.
+        - dest_bucket (str, optional): default destination bucket name; can be overwritten at
+            runtime. Required prior to running the task.
+        - dest_blob (str, optional): default destination blob name; can be
+            overwritten at runtime.  Required prior to running the task.
+        - project (str, optional): default Google Cloud project to work within; can be overwritten at runtime.
+            If not provided, will be inferred from your Google Cloud credentials
+        - credentials_secret (str, optional): the name of the Prefect Secret
+            which stores a JSON representation of your Google Cloud credentials; can be overwritten at runtime.
+            Defaults to `GOOGLE_APPLICATION_CREDENTIALS`.
+        - **kwargs (dict, optional): additional keyword arguments to pass to the Task constructor
+
+    Note that the design of this task allows you to initialize a _template_ with default settings.  Each inidividual
```

Comment:
Not that it's needed here, and feel free to mark as resolved once you see this, but a good example of this either in the examples folder or right here in this docstring is a flow where a blob is copied from A -> B and B -> C; having the kwargs overwritten at runtime.

Reply:
👍 added example